readme.md

```diff
@@ -192,7 +185,25 @@ A couple of examples:
 
 For additional guidance and important information about certificates, please refer to [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-changes-are-coming-and-why-you-should-care/ba-p/1658456) from the security team.
 
-## Long-Term Support (LTS)
+## Support
+
+The Azure IoT Hub Device Client supported releases is outlined in the following table.
+
+Refer to the [Azure IoT Device SDK lifecycle and support](https://learn.microsoft.com/azure/iot/iot-device-sdks-lifecycle-and-support) for details on the different supported stages.
+
+| Release | Category | End-of-life |
+|-|-|-|
+| 1.41.3 | Active | - |
+| 1.36.8 | Deprecated (LTS) | 2024-03-28 |
+| 2.0.0-preview004 | Preview | 2023-06-13
+
+### Need help?
```

Comment:
Blank lines around headers.

```suggestion
### Need help?

```

Reply:
https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines